wpas-hl7-translator/src/main/java/wales/nhs/dhcw/inthub/wpasHl7/servicebus/MessageReceiverClient.java

```diff
@@ -0,0 +1,44 @@
+package wales.nhs.dhcw.inthub.wpasHl7.servicebus;
```

Comment:
This class replicated the one from the Validator service...
I've extracted servicebus package classes to library subproject - message-bus-lib,
please merge/rebase against main branch and remove those classes